webapp-django/crashstats/crashstats/static/crashstats/css/components/panel.less

```diff
@@ -9,7 +9,7 @@
         .shadow-none();
     }
 
-    .title {
+    div.title {
```

Comment:
This applied to anything with class title. That doesn't work for the reST pages because the first h1 has class title and then shows up poorly. I figured it was easier to reduce the scope here.